app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,537 @@
+.index-group-lead-provider {
+  background: govuk-colour("light-grey");
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+
+
+.app-list--definition {
+  dt {
+    @include govuk-typography-weight-bold;
+  }
+
+  dd + dt {
+    margin-top: govuk-spacing(4);
+  }
+
+  dd {
+    margin-left: 0;
+  }
+
+  dd + dd {
+    margin-top: govuk-spacing(2);
+  }
+}
+
+.gem-c-contents-list__list, .gem-c-contents-list__nested-list {
```

Comment:
Stealing from govuk publishing components, I see you are a man of culture as well :D

Reply:
I have very little passion for recreating gov uk styles ;) Having said that - removed all that I havent used